### PR TITLE
ReblockGVCFs requires less memory for WGS samples when no interval list is provided

### PR DESCRIPTION
**Pull request by meganshand**
### Description

When a calling interval list is not provided to ReblockGVCFs, ValidateVariants takes a great deal of memory on WGS samples. When the WGS sample was processed with DRAGEN, there is no calling interval list, but DRAGEN skips some regions of the genome, so the reblocked GVCF does not validate unless it is using the input GVCF as the interval list. 

We fix the need for a large amount of memory in this case by first converting the GVCF input into an interval list (where abutting intervals are merged together). In the future, GATK might be able to solve this issue internally (see https://github.com/broadinstitute/gatk/issues/8608), but for now this workaround should fix the memory issue.

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [x] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

**Review by kayleemathews — APPROVED** [review]
Docs are updated!

**Review by nikellepetrillo — APPROVED** [review]
Looks good! i kicked of the scientific reblock tests off of this branch and they succeeded 🎉 https://gotc-jenkins.dsp-techops.broadinstitute.org/job/warp-workflow-tests/71142/console

**Review by jessicaway — APPROVED** [review]

**Comment by meganshand** [comment]
Thank you! @jessicaway Do I need one more review before this can be merged?

**Pull request merged**
